src/Resources/config/publish-workflow.xml

```diff
@@ -35,7 +35,7 @@
 
         <!-- integration with core security service -->
 
-        <service id="cmf_core.security.published_voter" class="Symfony\Cmf\Bundle\CoreBundle\PublishWorkflow\Voter\PublishableVoter" public="false">
+        <service id="cmf_core.security.published_voter" class="Symfony\Cmf\Bundle\CoreBundle\Security\Authorization\Voter\PublishedVoter" public="false">
```

Comment:
this voter needs the PublishWorkflowChecker service

Reply:
Yeah sorry, was still busy, didn't to run the tests locally